lib/services/ios-project-service.ts

```diff
@@ -860,7 +860,7 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 		return (() => {
 			let xcprojInfo = this.$xcprojService.getXcprojInfo().wait();
 			if (xcprojInfo.shouldUseXcproj && !xcprojInfo.xcprojAvailable) {
-				let errorMessage = `You are using CocoaPods version ${xcprojInfo.cocoapodVer} which does not support Xcode ${xcprojInfo.xcodeVersion.major}.${xcprojInfo.xcodeVersion.minor} yet.${EOL}In order for the NativeScript CLI to be able to work correctly with this setup you need to install xcproj command line tool and add it to your PATH. Xcproj can be installed with homebrew by running $ brew install xcproj from the terminal`;
+				let errorMessage = `You are using CocoaPods version ${xcprojInfo.cocoapodVer} which does not support Xcode ${xcprojInfo.xcodeVersion.major}.${xcprojInfo.xcodeVersion.minor} yet.${EOL}${EOL}You can update your cocoapods by running $sudo gem install cocoapods --pre from a terminal.${EOL}${EOL}In order for the NativeScript CLI to be able to work correctly with this setup you need to install xcproj command line tool and add it to your PATH. Xcproj can be installed with homebrew by running $ brew install xcproj from the terminal`;
```

Comment:
`--pre` flag is not required anymore. Cocoapods 1.0.0 is official, so there's no need to add `--pre` to the `sudo gem install`. This will install latest Cocoapods version, which is working fine with XCode 7.3.x